docs: updated readme files (#1955)
mondaycom/monday-ui-react-core: packages/core/README.md

## Storybook

We are using storybook in order to develop the components independently of any consumer.
run this to build & run the storybook locally:

```bash
npm run storybook
```

the storybook will be served on `http://localhost:7007`

## Developing locally with your consumer application

When developing locally, we are using a npm functionality called npm link, this allows us to
work locally on our package and use it in a different project without publishing.
This functionality basically overrides the npm mapping between package name to its repo, and points it to where the package is located locally.

### Troubleshooting local development

- If you are using NVM, make sure both packages are using the same version.
- Because we are using react hooks and having react as a peerDependency - if you want to develop locally and encounter issues with "invalid hook call" [see this github thread](https://github.com/facebook/react/issues/13991). The quick fix is in your webpack config file alias react to resolve the node_modules path

Go to the project's directory and run:

```zsh
nvm use
npm unlink
npm link
npm start
```

## Themes

> We support theming from the library to the component level using css variables - for more info on theming please read the [theme guidelines](./THEME_README.md) file

Reviewer: ```suggestion
Theming is supported using CSS variables - for more info on theming please read the [theme guidelines](./THEME_README.md) file
```

Reviewer: Don't we also support ThemeProvider now?

# Vibe Design System

[![npm](https://img.shields.io/npm/v/monday-ui-react-core)](https://www.npmjs.com/package/monday-ui-react-core)

![image](https://user-images.githubusercontent.com/60314759/147566893-63c5209a-8b83-4f32-af61-8b4c350ec770.png)

> [monday.com](https://www.monday.com) React components library - [Storybook](https://style.monday.com)

Reviewer: This should be in the root IMO

| --------- | --------- | --------- | --------- | --------- | --------- |
| last 4 versions| last 4 versions| 14+| last 2 versions| last 2 versions| last 2 versions |

## Installation

Install the component library

```
npm install monday-ui-react-core
```

## Usage

Components are imported from the library's root entry:

```javascript
import { Button } from "monday-ui-react-core";
```

In order to load all the relevant CSS tokens, you should import the `tokens` file at the root of your application file

```javascript
import "monday-ui-react-core/tokens";
```

> _If your project (or it's Storybook) is importing files differently - read more [here](DEPRECATED_IMPORTS.md)._

Reviewer: The link will be broken
